main/FatesConstantsMod.F90

```diff
@@ -30,11 +30,34 @@ module FatesConstantsMod
   ! Integer equivalent of false (in case come compilers dont auto convert)
   integer, parameter, public :: ifalse = 0
 
+  integer , parameter, public :: N_DBH_BINS = 6  ! no. of dbh bins used when comparing patches
```

Comment:
This seems like a better place for these constants

Reply:
I would like to go through (in a different PR) and gather the rest of these types of entities into either the constants or parameters modules...